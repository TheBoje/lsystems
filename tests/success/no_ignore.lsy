DERIVATION: 9;
ANGLE_FACTOR: 60.0;
SCALE_FACTOR: 1.0;
AXIOM:baaaaaaaaa;

PRODUCTIONS:
    b < a > * --> b;
    * < b > * --> a;
END_PRODUCTIONS;
