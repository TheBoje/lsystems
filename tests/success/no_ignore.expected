node: b < a > * --> b
node: * < b > * --> a
config: 
	derivation: 9
	angle factor: 60
	scale factor: 1
	axiom: baaaaaaaaa
	ignore: none

derivation 0/9: baaaaaaaaa
derivation 1/9: abaaaaaaaa
derivation 2/9: aabaaaaaaa
derivation 3/9: aaabaaaaaa
derivation 4/9: aaaabaaaaa
derivation 5/9: aaaaabaaaa
derivation 6/9: aaaaaabaaa
derivation 7/9: aaaaaaabaa
derivation 8/9: aaaaaaaaba
derivation 9/9: aaaaaaaaab
aaaaaaaaab
